lightning/src/routing/router.rs

```diff
@@ -784,6 +785,13 @@ where L::Target: Logger {
 					// Verify the liquidity offered by this channel complies to the minimal contribution.
 					let contributes_sufficient_value = available_value_contribution_msat >= minimal_value_contribution_msat;
 
+
+					// Do not consider candidates that exceed the MAX_TOTAL_CLTV_DELTA limit.
+					const MAX_TOTAL_CLTV_DELTA: u32 = 1000;
+					let hop_total_cltv_delta = $next_hops_cltv_delta + $directional_info.cltv_expiry_delta as u32;
+					let doesnt_exceed_cltv_delta_limit = hop_total_cltv_delta <= MAX_TOTAL_CLTV_DELTA;
+					log_trace!(logger, "Checking total CLTV delta: {} / {}. doesnt_exceed_cltv_delta_limit: {}", hop_total_cltv_delta, MAX_TOTAL_CLTV_DELTA, doesnt_exceed_cltv_delta_limit);
```

Comment:
Hmm, we currently don't log *this* verbosely even at TRACE. Honestly we should probablve move that direciton, but for now let's not make TRACE fill GBs if you're calculating routes.

Reply:
I totally agree. I just left that in there for now for debugging purposes. Will remove it before eventually squashing.